build.d/300-oca-dependencies

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+
+import os
+import yaml
+import glob
+import urllib2
+
+from odoobaselib import CUSTOM_DIR, ADDONS_DIR
+
+
+# Method from maintainer-quality-tools
+def parse_depfile(depfile, owner='OCA'):
+    deps = []
+    for line in depfile:
+        line = line.strip()
+        if not line or line.startswith('#'):
+            continue
+        parts = line.split()
+        repo = parts[0]
+        if len(parts) > 2:
+            branch = parts[2]
+        else:
+            branch = os.environ.get('VERSION', '10.0')
```

Comment:
This env var is called `ODOO_VERSION`, and you should not default to any value. It is considered an error if it is not available. Just use brackets and let the proper `KeyError` bubble up if it ever happens.